fix: accurate toogleAllSelection discription

HomyeeKing opened the pull request:
Please make sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure you follow Element's contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
* [x] Make sure you are merging your commits to `dev` branch.
* [x] Add some descriptions and refer to relative issues for your PR.

精确一下`toogleAllSelection`方法的描述， 应该是 切换全选和全不选
之前的听起来像反选，但实际表现是全选和全不选

JeremyWuuuuu commented:
Please do update all languages.

HomyeeKing commented:
> Please do update all languages.

em..  it's better leave it to the native speaker 👀

zazzaz commented:
Note, we will update the English and Chinese document by ourself. And copy the English version to other languages if not familiar with. Then leave to the native speaker to translate.

HomyeeKing committed:
Update table.md

Update table.md

accurate  `toogleAllSelection` description for the former one sounds like `reverse` row selected state

Update table.md

精确一下`toogleAllSelection`方法的描述， 应该是 切换全选和全不选

之前的听起来像反选，但实际表现是全选和全不选

Update table.md

Update table.md (#5)

fix: doc

Update table.md

HomyeeKing commented:
done

zazzaz commented on website/docs/es/table.md:
```diff
@@ -1878,7 +1878,7 @@ Puede personalizar el índice de la fila con la propiedad `type=index` de las co
 | clearSelection     | utilizado en selección múltiple de la tabla, limpiar selección | —     |
 | toggleRowSelection | utilizado en selección múltiple de la tabla, alterna si una cierta fila es seleccionada. Con el segundo parámetro, puede directamente establecer si la fila es seleccionada | row, selected |
-| toggleAllSelection | usado en Table de selección múltiple, cambia los estados de selección de todas las filas. | - |
+| toggleAllSelection | used in multiple selection Table, toggle select all and deselect all | - |
```
I'd like to see the left part before the comma to keep the original language, please.

zazzaz commented on website/docs/fr-FR/table.md:
```diff
@@ -1882,7 +1882,7 @@ Vous pouvez personnaliser les indices des colonnes de type `index`.
 | clearSelection | Dans les tables avec sélection multiple, efface la sélection. | — |
 | toggleRowSelection | Dans les tables avec sélection multiple, change la sélection d'une ligne. Grâce au deuxième paramètre vous pouvez directement décider si cette ligne est sélectionnée. | row, selected |
-| toggleAllSelection | Utilisé dans les tables à sélection multiples, sélectionne toutes les lignes. | - |
+| toggleAllSelection | used in multiple selection Table, toggle select all and deselect all | - |
```
And this line too, please

HomyeeKing replied:
oops..  really embarrassing 😅

HomyeeKing commented:
@zazzaz  is there any suggestions about this
